libnmstate/schemas/operational-state.yaml

```diff
@@ -408,3 +416,49 @@ definitions:
                     type: boolean
                   state:
                     type: string
+  rib:
+    type: object
+    properties:
+      name:
+        type: string
+      route-table-id:
+        type: integer
+      address-family:
+        type: string
+        enum:
+          - ipv4
+          - ipv6
+      default-rib:
```

Comment:
I do not think we need this one at the moment... unless changing it will have a special affect.
It has relevance to the ip rules, but I am not clear how we can link this to that at the moment.

Reply:
Removed.